### DISCUSSION TITLE
Not able to merge Pull Request

### DISCUSSION THREAD

**Discussion by HamzaMeh** [ByteLegendQuest/remember-brave-people#773] [Q&A] [answered]
Getting this error on opening Pull Request for Notice board:

Error: Exception in thread "main" java.lang.IllegalStateException: Push failed. There might be multiple jobs running simultaneously, try rerunning the job.
error: failed to push some refs to 'https://github.com/ByteLegendQuest/remember-brave-people.git'
	at com.bytelegend.game.Git.push(Git.java:87)
	at com.bytelegend.game.CIDataGeneratorJob.run(CIDataGeneratorJob.java:84)
	at com.bytelegend.game.CIDataGeneratorJob.main(CIDataGeneratorJob.java:51)
Error: Process completed with exit code 1

**Comment by blindpirate** [comment, marked as answer]
This is merged. Thanks for reporting.

https://github.com/ByteLegendQuest/remember-brave-people/pull/772
